### PR TITLE
[HMA-3448] Change duplicate header content

### PR DESCRIPTION
**Pull request by mrolfe88**
Remove back button from nav landmark

### CODE REVIEW

File: app/views/pages/ResultDetailView.scala.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 <br/>
 
-@h2(Messages("quick_calc.result.total_deductions.info"))
+@h2(Messages("quick_calc.result.total_deductions.info_a") + " " + label.toLowerCase + "ly " + Messages("quick_calc.result.total_deductions.info_b"))
```
Comment (georgeherby) [inline app/views/pages/ResultDetailView.scala.html:42]: Pretty sure there are labels with ly. ALSO you need this in a message file to we can support welsh

File: app/views/pages/ResultDetailView.scala.html [hunk 2]
```diff
@@ -85,7 +85,7 @@
         @Messages("quick_calc.about_k_code_c")
     }
 
-    @h2(Messages("quick_calc.result.how_we_calculated"))
+    @h2(Messages("quick_calc.result.how_we_calculate_a") + " " + label.toLowerCase + "ly " + Messages("quick_calc.result.how_we_calculate_b"))
```
Comment (georgeherby) [inline app/views/pages/ResultDetailView.scala.html:88]: Same here

File: app/views/pages/ResultDetailView.scala.html [hunk 3]
```diff
@@ -129,7 +129,7 @@
     )
 } else {
 
-    @h2(Messages("quick_calc.result.how_we_calculated"))
+    @h2(Messages("quick_calc.result.how_we_calculate_a") + " " + label.toLowerCase + "ly " + Messages("quick_calc.result.how_we_calculate_b"))
```
Comment (georgeherby) [inline app/views/pages/ResultDetailView.scala.html:132]: same here

**Review by georgeherby — CHANGES_REQUESTED** [review]

**Comment by mrolfe88** [comment]
test this please

**Comment by mrolfe88** [comment]
test this please

**Pull request merged**